src/graphql/entries.ts

```diff
@@ -63,34 +67,90 @@ function updateRichTextField(
   locale: string
 ) {
   if (name in modified) {
+    if (!modified[name]) {
+      modified[name] = {};
+    }
     (modified[name] as { json: unknown }).json = update?.fields?.[name]?.[locale] ?? null;
```

Comment:
would we still need the typecast if we define the type it in L71?

Reply:
unfortunately yes 😢